src/api/applicant/content-types/applicant/schema.json

```diff
@@ -66,6 +66,12 @@
       "type": "component",
       "repeatable": true,
       "component": "skills.skills"
+    },
+    "yearsExperience": {
+      "type": "decimal",
+      "required": true,
+      "min": 0,
+      "max": 100
```

Comment:
Hope humanity will find a lot of cure soon to fight aging and we will need to update this limit even further 😄